test/tests/silverpeas-basics/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+cname="silverpeas-container-$RANDOM-$RANDOM"
+
+# when running the first time, a silverpeas process is spawn before starting Silverpeas
+# (this configuration process can take some time)
+cid="$(docker run -d -e DB_SERVERTYPE="H2" -e DB_SERVER=":file:" -e DB_PASSWORD="sa" --name "$cname" "$image")"
+trap "docker rm -vf $cid > /dev/null" EXIT
+
+# the maximum time for Silverpeas to be configured and to be started (if no errors)
+sleep 120s
```

Comment:
I'm a little concerned about this -- this will make the test run for 2 minutes every time regardless of how fast silverpeas actually starts up.  Any way this could be converted to use `retry.sh` like some of the other tests do?  I'm happy to help on this front if a solution doesn't seem obvious to you at first glance. :+1: